setup.cfg

```diff
@@ -17,7 +17,7 @@ packages = find:
 install_requires =
     ansible-core
     ansible-builder>=1.0.1,<2.0
-    ansible-lint>=5.0.8,<6.0
+    ansible-lint>=6.0.0,<7.0
```

Comment:
Why the jump to `>=6.0` and not leave `>=5.0` ? Any technical reason ?

Reply:
They diverged considerably and will produce different results. We can keep it like this but you should be aware that ansible-lint upstream will not provide support for 5.x.

Ideally, you should look into removing support for py37.